app/src/main/java/com/scouts/kitchenplaner/hilt/DataLayerModule.kt

```diff
@@ -42,6 +43,11 @@ class DataLayerModule {
         return database.shoppingListDao()
     }
 
+    @Provides
+    fun recipeDao(database: KitchenAppDatabase): RecipeDAO {
```

Comment:
rename to provideRecipeDao

(other @Provides methods should be renamed in the ProjectOverviewDataLayer branch)